demo/index.html

```diff
@@ -14,12 +14,12 @@
     }
   </style>
   <script src="http://rawgit.com/exif-js/exif-js/master/exif.js"></script>
-  <script src="http://rawgit.com/buunguyen/exif-orient/master/exif-orient.js"></script>
+  <script src="../exif-orient.js"></script>
```

Comment:
Have you tested that this change work when accessed via http://rawgit.com/YOUR-NAME/exif-orient/master/demo/index.html?